src/image-upload/image-upload.component.ts

```diff
@@ -26,6 +26,7 @@ export class ImageUploadComponent implements OnInit {
 
   @Input() beforeUpload: (UploadMetadata) => UploadMetadata | Promise<UploadMetadata> = data => data;
   @Input() buttonCaption: string = 'Select Images';
+  @Input('class') klass: string = 'img-ul';
```

Comment:
Okay @UncleDave i will cahnge it to `cssClass`. Sure i will add demo for the same.